src/agreementManager/AgreementManager.ts

```diff
@@ -324,58 +323,25 @@ class AgreementManager {
         return undefined;
     }
 
-    /**
-     * Check if a participant has signed a dispute
-     */
-    public hasParticipantSignedDispute(
-        dispute: DisputeStruct,
-        participant: Address
-    ): boolean {
-        const disputeHash = ethers.keccak256(
-            Codec.encode(dispute, Type.Dispute)
-        );
-        const disputeConfirmation =
-            this.storage.disputes.getDisputeConfirmation(disputeHash);
-
-        if (!disputeConfirmation) return false;
-
-        // Check if participant is the disputer
-        if (dispute.input.disputer === participant) return true;
-
-        // Check confirmation signatures
-        for (const sig of disputeConfirmation.signatures) {
-            const signer = ethers.verifyMessage(
-                ethers.getBytes(disputeHash),
-                sig as Signature
-            );
-            if (signer === participant) return true;
-        }
-
-        return false;
-    }
-
-    public async getForkDisputeConfirmations(
```

Comment:
This code is not used currently and we're skipping it for now to ship faster, but I'd leave it here with DisputeConfriamtions since it was already implemented and a `DisputeConfirmation` is a super set of `Dispute` and you can easily go from `DisputeConfirmation` to `Dispute` and vice versa - solidity code accounted for finality on this and parts of the event pipeline also account for this - we just didn't finish the implementation which we'll eventually finish